randovania/games/prime1/hint_distributor.py

```diff
@@ -1,12 +1,19 @@
 from __future__ import annotations
 
-from typing import override
+from typing import TYPE_CHECKING, override
 
 from randovania.game_description.hint import HintItemPrecision, HintLocationPrecision, PrecisionPair
 from randovania.generator.hint_distributor import HintDistributor
 
+if TYPE_CHECKING:
+    from randovania.game_description.pickup.pickup_entry import PickupEntry
+
 
 class PrimeHintDistributor(HintDistributor):
+    @override
+    def is_pickup_interesting(self, pickup: PickupEntry) -> bool:
+        return not pickup.has_hint_feature("key")
```

Comment:
using the `"artifact"` feature here might make it more obvious what's being excluded

Reply:
Something that may be worth discussing in the future is whether we want the key feature to still exist in the first place.

AFAIK the previous justification was for it to introduce ambiguitiy between whether that's the echoes' STK/DTK or the prime's Artifact. But with how early prime gets artifact hints in like 90% of cases one could also argue that it's not really doing anything